README.md

```diff
@@ -1,3 +1,82 @@
 # pay-cli
 
 This project provides CLI for Pay developers to interact with GOV.UK Pay's API, CI/CD platform and internal tooling.
+
+## Deployer Command
+This is a command help automate the rotation of the `ci.deployer` AWS API key and secret in each of our environments.
+
+As the process requires us to rotate the keys in one environment and then store the new ones using credstash in either of the CI or Deploy AWS Environments, the best way to do this is with a script to safely rotate the keys whilst minimising any opportunity for expired or invalid keys to be used.
+
+### Requirements
+Before using this script, you will need the following things to be installed:
+
+* aws-vault
+* credstash
+* ykman
+
+You will also need access to the necessary AWS accounts for the credentials you are rotating, e.g. `govuk-pay-dev` and `govuk-pay-test` require access to `govuk-pay-ci`, and `govuk-pay-staging` and `govuk-pay-prod` require access to `govuk-pay-deploy`.
+
+### Usage
+At the current time of writing, this is the usage of the script:
+
+```sh
+./pay deployer --environment [dev/test/staging/production] --management-profile [ci/deploy]
```

Comment:
`deployer` doesn't conjure up images of rotating the aws api key for `ci.deployer` user but I'm struggling to think of something more suitable that's practical. The readme and help text make it clear so 👍